DataConnectors/OracleDatabaseAudit/Connector_OracleDatabaseAudit.json

```diff
@@ -0,0 +1,114 @@
+{
+    "id": "OracleDatabaseAudit",
+    "title": "Oracle Database Audit",
+    "publisher": "Oracle",
+    "descriptionMarkdown": "The Oracle DB Audit data connector provides the capability to ingest [Oracle Database](https://www.oracle.com/database/technologies/) audit events into Azure Sentinel through the syslog. Refer to [documentation](https://docs.oracle.com/en/database/oracle/oracle-database/21/dbseg/introduction-to-auditing.html#GUID-94381464-53A3-421B-8F13-BD171C867405) for more information.",
+    "additionalRequirementBanner": "This data connector depends on a parser based on Kusto Function to work as expected. Follow the steps to use this Kusto Function alias **OracleDatabaseAuditEvent** in queries and workbooks. [Follow steps to get this Kusto Function>](https://aka.ms/sentinel-OracleDatabaseAudit-parser)",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "Syslog(OracleDatabaseAudit)",
+            "baseQuery": "OracleDatabaseAuditEvent"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "Top 10 Sources",
+            "query": "OracleDatabaseAuditEvent\n | summarize count() by SrcDvcHostname\n | top 10 by count_"
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "Syslog(OracleDatabaseAudit)",
```

Comment:
Please keep the space next to Syslog, There should not be any space within "OracleDatabaseAudit.